File: GanttChartLightLibraryDemos/Demos/Samples/WPF-CSharp/GanttChartDataGrid/AssignmentsTemplate/CustomGanttChartItem.cs
using System;
using System.Windows.Media;
using DlhSoft.Windows.Controls;
using System.ComponentModel;
using System.Collections.ObjectModel;
using System.Collections.Specialized;
using System.Collections.Generic;
using System.Linq;

namespace Demos.WPF.CSharp.GanttChartDataGrid.AssignmentsTemplate
{
    class CustomGanttChartItem : GanttChartItem
    {
        private ImageSource assignmentsIconSource;
        public ImageSource AssignmentsIconSource
        {
            get { return assignmentsIconSource; }
            set
            {
                assignmentsIconSource = value;
            }
        }
    }
}

File: GanttChartLightLibraryDemos/Demos/Samples.Resources/WPF-VisualBasic/GanttChartDataGrid/AssignmentsTemplate/CustomGanttChartItem.vb
Imports DlhSoft.Windows.Controls
Imports System.ComponentModel
Imports System.Collections.ObjectModel
Imports System.Collections.Specialized

Friend Class CustomGanttChartItem
    Inherits GanttChartItem

    Private assignmentsIconSourceValue As ImageSource
    Public Property AssignmentsIconSource() As ImageSource
        Get
            Return assignmentsIconSourceValue
        End Get
        Set(value As ImageSource)
            assignmentsIconSourceValue = value
        End Set
    End Property
End Class
